akka-stream/src/main/scala/akka/stream/impl/Stages.scala

```diff
@@ -76,6 +76,7 @@ import akka.stream.Attributes._
     val merge = name("merge")
     val mergePreferred = name("mergePreferred")
     val mergePrioritized = name("mergePrioritized")
+    val mergePrioritizedN = name("mergePrioritizedN")
```

Comment:
I think the default `mergePrioritized` name (set in `MergePrioritized`) is enough and that this is just a convenience factory, so skip this.